### ISSUE TITLE
Route_access_denied when posting on the list_comments route

### ISSUE THREAD

**Issue by Schimpf-Christian** [rpeyron/dropbox-comments-downloader#1]
Thank you very much for sharing the undocumented API route with us.
However, when I try post on the "2/comments2/list_comments" from my Android mobile app, I receive a "route_access_denied" error. Is it the same for you? I could imagin that Dropbox blocks the route after noticing that people are actually use it unintentionally. I just wanted to ask if you experience the same problems now.

**Comment by rpeyron** [comment]
Hello, unfortunately yes it is the same for me ; I tested it today, and it is not working anymore.
I have looked again at the Dropbox application, and it seems it uses other magic to show comments, but no more this API.
I am not sure if this is intentional from Dropbox, but as it was an undocumented API it is not abnormal that they removed it if they didn't have the use for it.  Hope they will soon release an official supported API for comments...
Thanks for reporting, I have added a notice in the README about that.

**Comment by Schimpf-Christian** [comment]
Yes I also hope that, because it's really missing. People are bugging them since 2017 to get a comment support in the API, for any reason they are very reluctant in this regard.
